src/bin/hamcore/strtable_ru.stb

```diff
@@ -7122,7 +7122,7 @@ CMD_TrafficClient_ERROR_HOSTPORT	The host name or port number is incorrectly spe
 # TrafficServer command
 CMD_TrafficServer		Запустить средство тестирования скорости сетевого трафика в режиме сервера
 CMD_TrafficServer_Help	Используется для запуска инструмента измерения пропускной способности в режиме сервера. \nДве команды, TrafficClient и TrafficServer, используются для измерения пропускной способности между двумя компьютерами, соединенными сетью IP. \nУкажите номер порта и запустите серверную часть с помощью команды TrafficServer, чтобы прослушивать подключение от TrafficClient другого компьютера. \nВы можете отобразить более подробную информацию об инструменте измерения пропускной способности, введя "TrafficClient ?". \n\nПримечание. Эту команду можно вызвать из утилиты управления командной строкой SoftEther VPN. Вы также можете выполнить эту команду при подключении к текущему VPN-серверу или VPN клиенту в режиме администрирования, но фактически осуществляет связь и измеряет пропускную способность тот компьютер, на котором выполняется команда, а не компьютер с которого выполнено подключение в режиме администрирования.
-CMD_TrafficServer_Args	TrafficServer [порт] [/NOHUP:да|нет]
+CMD_TrafficServer_Args	TrafficServer [port] [/NOHUP:да|нет]

```

Comment:
I think this should be either of the following.
* `CMD_TrafficServer_Args	TrafficServer [порт] [/NOHUP:да|нет]`
* `CMD_TrafficServer_Args	TrafficServer [port] [/NOHUP:yes|no]`

Reply:
I replaced it with `CMD_TrafficServer_Args TrafficServer [port] [/NOHUP:yes|no]`.

Reply:
Since NOHUP parameter is determined by whether it is a string starting with "y", it should be written as "yes" in the legend.
https://github.com/SoftEtherVPN/SoftEtherVPN/blob/18dc2621ec893509c96fa6e97e5d64ca7d149566/src/Cedar/Console.c#L1443